data/transition-sites/gds_alphagov_alert.yml

```diff
@@ -0,0 +1,10 @@
+---
+site: gds_alphagov_alert
+whitehall_slug: government-digital-service
+homepage: https://alert.publishing.service.gov.uk/
```

Comment:
As @jamiecobbett [pointed out](https://github.com/alphagov/transition-config/pull/1111#discussion_r34457874) on the other PR, Bouncer [uses `File.join`](https://github.com/alphagov/bouncer/blob/master/lib/bouncer/outcome/global_type.rb#L7-L9) to join the host with the path for global redirects, so in this case the trailing slash on the global new URL won't put a double slash in the `Location` URL:

```
$ curl -I -H 'Host: alert.production.alphagov.co.uk' 'http://redirector-cdn.production.govuk.service.gov.uk/test/path'
HTTP/1.1 301 Moved Permanently
Location: https://alert.publishing.service.gov.uk/test/path
```

This only applies to global redirects appending path, though - all normal global redirects and all homepage redirects will use the URL as it appears in the config, so will have a single trailing slash if there's one here.

It does look a bit messy in Transition and on those redirect URLs, so it would be nice to standardise on no trailing slashes:
![screen shot 2015-08-17 at 15 57 32](https://cloud.githubusercontent.com/assets/1822424/9307857/be90ffbe-44f8-11e5-891b-757fb1c8cd81.png)